crates/sui-benchmark/src/lib.rs

```diff
@@ -557,7 +557,7 @@ impl ValidatorProxy for FullNodeProxy {
                 .await
             {
                 Ok(resp) => {
-                    let effects = ExecutionEffects::SuiTransactionEffects(resp.effects);
+                    let effects = ExecutionEffects::SuiTransactionEffects(resp.effects.unwrap());
```

Comment:
I am treating `benchmarking` as a test, so the usage of `unwrap` here should be fine? cc @sadhansood

Reply:
This seems ok to me. I'd maybe use an `expect()` with a custom and helpful error message instead.